app/models/form/page.rb

```diff
@@ -1,24 +1,21 @@
 class Form::Page
-  attr_accessor :id, :header, :description, :questions, :soft_validations,
-                :depends_on, :subsection, :hide_subsection_label
+  attr_accessor :id, :header, :description, :questions,
+                :depends_on, :title_text, :informative_text, :subsection, :hide_subsection_label
 
   def initialize(id, hsh, subsection)
     @id = id
     @header = hsh["header"]
     @description = hsh["description"]
     @questions = hsh["questions"].map { |q_id, q| Form::Question.new(q_id, q, self) }
     @depends_on = hsh["depends_on"]
+    @title_text = hsh["title_text"]
+    @informative_text = hsh["informative_text"]
     @hide_subsection_label = hsh["hide_subsection_label"]
-    @soft_validations = hsh["soft_validations"]&.map { |sv_id, s| Form::Question.new(sv_id, s, self) }
     @subsection = subsection
   end
 
   def expected_responses
```

Comment:
Is this method still needed at all? If it's just a wrapper for questions now can probably use `questions` directly wherever this is used?

Reply:
Probably not - we're still in the process of removing all the unneeded bits and tidying up the resulting code. We'll probably end up removing the method 😄